Integrations/integration-Quinc.yml

```diff
@@ -0,0 +1,341 @@
+        data = {
+            'ips': { 'targets' : [args['targetIP']]},
+            'MemoryAcquistion' : { 'Operation': 11 }
+        }
+        res = send_request('POST',  QUINC_URL + url, json.dumps(data))
+        ec = {
+            'ID': res,
+            'Type': 'LegacyMemoryDump'
+        }
+        return {
+            'Type' : entryTypes['note'],
+            'ContentsFormat' : formats['json'],
+            'Contents' : ec,
+            'HumanReadable': "JobID: " + str(res),
+            'EntryContext': { JOB_KEY+'(val.ID && val.Type == obj.Type)' : ec }
+        }
+
+    def quinc_read_casefile_command(args):
+        url = 'api/v2/enterpriseapi/core/readfilecontents';
+        data = args['filepath']
+        res = send_request('POST',  QUINC_URL + url, json.dumps(data))
+        return {
+            'Type' : entryTypes['note'],
+            'ContentsFormat' : formats['text'],
+            'Contents' : res,
+            'EntryContext': { 'Accessdata.File.Contents' : res }
+        }
+
+    # The command demisto.command() holds the command sent from the user.
+    if demisto.command() == 'test-module':
+        # This is the call made when pressing the integration test button.
+        demisto.results(SendQuincTestRequest())
+        sys.exit(0)
+    if demisto.command() == 'accessdata-legacyagent-get-software':
+        demisto.results(QuincLegacyAgentSoftwareInventory(demisto.args()))
+        sys.exit(0)
+    if demisto.command() == 'accessdata-get-jobstatus':
+        demisto.results(quinc_get_jobstatus_command(demisto.args()))
+        sys.exit(0)
+    if demisto.command() == 'accessdata-legacyagent-get-processlist':
+        demisto.results(quinc_legacyagent_runvolatilejob_command(demisto.args()))
+        sys.exit(0)
+    if demisto.command() == 'accessdata-legacyagent-get-memorydump':
+        demisto.results(quinc_legacyagent_runmemoryacquisition_command(demisto.args()))
+        sys.exit(0)
+    if demisto.command() == 'accessdata-read-casefile':
+        demisto.results(quinc_read_casefile_command(demisto.args()))
+        sys.exit(0)
+
+    if demisto.command() == 'fetch-incidents':
+        # You can store the last run time...
+        demisto.setLastRun({'time': 'now'})
+        # And retrieve it for use later:
+        lastRun = demisto.getLastRun()
+        # lastRun is a dictionary, with value "now" for key "time".
+        # JSON of the incident type created by this integration
+        demisto.incidents([{"Name":"Incident #1"},{"Name":"Incident #2"}])
+        sys.exit(0)
+    # You can use demisto.args()[argName] to get a specific arg. args are strings.
+    # You can use demisto.params()[paramName] to get a specific params.
+    # Params are of the type given in the integration page creation.
+  type: python
+  commands:
+  - name: accessdata-legacyagent-get-software
+    arguments:
+    - name: caseid
+      required: true
+      description: ID of case
+    - name: targetIP
```

Comment:
the conventions are:
targetIP -> target_ip

Reply:
code conventions README
https://github.com/demisto/content/blob/master/docs/code_conventions/README.MD